changelog.md

```diff
@@ -277,6 +277,7 @@
 - Tuple expressions are now parsed consistently as
   `nnkTupleConstr` node. Will affect macros expecting nodes to be of `nnkPar`.
 
+- `nimscript` now accepts arbitrary file extensions.
```

Comment:
can you add a test? (to avoid https://github.com/nim-lang/Nim/pull/17579#discussion_r603825363)

you could simply add 1 block near this:
```
  block: # config.nims, nim.cfg, hintConf, bug #16557
    let cmd = fmt"{nim} r {defaultHintsOff} --hint:conf tests/newconfig/bar/mfoo.nim"
    let (outp, exitCode) = execCmdEx(cmd, options = {poStdErrToStdOut})
```
in trunner.nim

Reply:
well 1 is better than 0... and might be enough

Reply:
yeah, I'm preparing this test.